src/app/datagrid/selection/selection.ts

```diff
@@ -31,10 +31,27 @@ export class DatagridSelectionDemo {
     example = EXAMPLE;
     users: User[];
     selected: User[] = [];
+    toAdd: User[] = [];
+    toDelete: User;
+    toEdit: User;
 
     constructor(private inventory: Inventory) {
         inventory.size = 10;
         inventory.reset();
         this.users = inventory.all;
     }
+
+    onDelete() {
+        this.toDelete = this.selected[0];
```

Comment:
I treated these demo components more like playground with incomplete details (as I am going to break up to several demo components for the website). But I updated it just for the sake of completeness.